jarbas_hive_mind/__main__.py

```diff
@@ -0,0 +1,27 @@
+from jarbas_hive_mind import get_listener
+from jarbas_hive_mind.configuration import CONFIGURATION
+
+def main():
+    import argparse
+    parser = argparse.ArgumentParser(description='Start HiveMind as a server')
+    parser.add_argument("--name", help="human readable name")
+    parser.add_argument("--access_key", help="access key")
+    parser.add_argument("--crypto_key", help="payload encryption key")
+    parser.add_argument("--mail", help="human readable mail")
```

Comment:
mail is currently unused and a placeholder, will likely be removed in the future, don't include it

Reply:
Flags crypto_key and mail are optional, they can be removed in the future if they are not being used.

Reply:
crypto_key should remain available and optional, but mail is not used and i dont think it will ever be so should be removed for now

Reply:
Mail eliminated from the script and from the database in the last commit.